.travis.yml

```diff
@@ -95,8 +95,10 @@ jobs:
         - export CONDA_PATH="\$env:Path = ';C:\miniconda3;C:\miniconda3\Library\mingw-w64\bin;C:\miniconda3\Library\usr\bin;C:\miniconda3\Library\bin;C:\miniconda3\Scripts;C:\miniconda3\bin;C:\miniconda3\condabin;C:\iverilog\bin;' + \$env:Path"
         - powershell "$CONDA_PATH; conda install --yes -c msys2 m2-base m2-make m2w64-toolchain libpython"
         - powershell "$CONDA_PATH; conda install --yes virtualenv"
+        - powershell "$CONDA_PATH; conda update --yes pip"
+        - powershell "$CONDA_PATH; conda info; conda list --show-channel-urls"
         # needed to make the builtin sqlite3 module (used by coverage) work - see gh-1909
-        - powershell '$CONDA_PATH; cp C:\miniconda3\Library\bin\sqlite3.dll C:\miniconda3\DLLs\'
+        - powershell "dir C:\miniconda3\Library\bin; dir C:\miniconda3\DLLs; C:\miniconda3; cp C:\miniconda3\Library\bin\sqlite3.dll C:\miniconda3\DLLs\; exit 0"
```

Comment:
Was your hope that newer versions would fix this?

Reply:
No, the command failed in a travis run, and I'm not sure yet why.

Reply:
Seems to be a quoting issue, let's see.